includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -1556,11 +1557,34 @@ public function update_subscription_item( $wcpay_subscription_item_id, $data ) {
 	 * @throws API_Exception If payment method does not exist.
 	 */
 	public function get_payment_method( $payment_method_id ) {
-		return $this->request(
+		// A fallback for sites, which migrated from Stripe to WCPay.
+		$is_source = preg_match( '/^src_\w+$/i', $payment_method_id );
+		if ( $is_source ) {
+			$api = self::SOURCES_API;
+		} else {
+			$api = self::PAYMENT_METHODS_API;
+		}
+
+		$response = $this->request(
 			[],
-			self::PAYMENT_METHODS_API . '/' . $payment_method_id,
+			$api . '/' . $payment_method_id,
 			self::GET
 		);
+
+		if ( $is_source ) {
```

Comment:
Let's use early return to reduce nesting level.
```suggestion
		if ( ! $is_source ) {
		    return $response;
		}
```

Reply:
Done in https://github.com/Automattic/woocommerce-payments/pull/4663/commits/0ad203a85bf1c300d838519ab3eeec2d9e39dbd1 👍